src/main/java/utility/FileHandler.java

```diff
@@ -0,0 +1,68 @@
+package utility;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
+
+import exception.GenesisException;
+
+public class FileHandler {
+    final static String DIR_PATH = "./store";
+    final static String FILE_PATH = "./store/genesis.txt";
```

Comment:
Stretch goal, use a config to specify where the file should be rather than hard coding.